src/api.raml

```diff
@@ -1126,15 +1126,15 @@ traits:
                     description: |
                         Press package.
                     body:
-                        application/vnd.elife.press-package+json;version=1:
-                            schema: !include ../dist/model/press-package.v1.json
+                        application/vnd.elife.press-package+json;version=2:
```

Comment:
We need to leave in the version 1 as we'll support it for a period of time (as least the duration of the switchover).